server/settings.py

```diff
@@ -229,6 +229,10 @@ def env(variable, fallback_value=None):
         'thumbnail': {'width': 220, 'height': 120},
         'viewImage': {'width': 640, 'height': 640},
         'baseImage': {'width': 1400, 'height': 1400},
+
+        'preview': {'width': 1024, 'height': 768},
```

Comment:
so in the ui you can only edit those from vocabularies, so maybe we should show there also those system defined ones, I can handle it there, it would make sense to be editable imo